util/random.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
You make it a separate package, but not test for it, our coverage down continuous.